src/core/config-schema.helper.ts

```diff
@@ -0,0 +1,125 @@
+import * as Joi from '@hapi/joi';
+import chalk from 'chalk';
+
+const schema = Joi.object({
+  browserWidth: Joi.number()
+    .min(100)
+    .max(2000)
+    .required(),
+  browserHeight: Joi.number()
+    .min(100)
+    .max(2000)
+    .required(),
+  browserLanguage: Joi.string().required(),
```

Comment:
Maybe we should validate if the given browserLanguage exists? e.g. `en-GB`

Reply:
@sethii what do you think?

Reply:
I wouldn't do so, it will just complicate things unecessary

Reply:
I've checked what will happen if I provide e.g ```browserLanguage: 'knight'``` our tests start normally but ```navigator. language``` will be ```knight```  and if the user will open ```chrome://settings/languages``` there will be no default language. 

![Screen Shot 2019-11-27 at 8 47 45 AM](https://user-images.githubusercontent.com/32454314/69703834-9966c280-10f2-11ea-8609-1ec33de9ba10.png)

I've also added in separated file object with country and countryCodes. And finally it would like that: ```browserLanguage: Joi.string().valid(...code)```

But as @sethii mentioned - we don't want to complicate it.